Packs/Slackbot_V2/Integrations/SlackBot/SlackBot.yml

```diff
@@ -0,0 +1,88 @@
+category: Messaging
+commonfields:
+  id: SlackBot
+  version: -1
+configuration:
+- display: Long running instance
+  name: longRunning
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: ''
+  displaypassword: Bot Token
+  hiddenusername: true
+  name: bot_token
+  required: true
+  type: 9
+- display: ''
+  displaypassword: App Token
+  hiddenusername: true
+  name: app_token
+  required: true
+  type: 9
+- additionalinfo: Base XSOAR URL path
+  display: XSOAR Base URL
+  name: base_url
+  required: true
+  type: 0
+- display: ''
+  displaypassword: XSOAR API Key
+  hiddenusername: true
+  name: api_key
+  required: true
+  type: 9
+- display: XSOAR API Key ID
+  name: api_key_id
+  required: false
+  type: 0
+- additionalinfo: Dedicated Slack channel to receive notifications
+  display: Dedicated Slack Notification Channel
+  name: dedicated_slack_channel_name
+  required: false
+  type: 0
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Comma separated roles that are allowed to execute the slash commands
+  defaultvalue: Instance Administrator,Account Admin
+  display: Allowed roles
+  name: roles
+  required: false
+  type: 0
+contentitemexportablefields:
+  contentitemfields:
+    definitionid: ''
+    fromServerVersion: ''
+    itemVersion: ''
+    packID: dfa47260-fa41-4dc4-80a9-54d34feb3ed0
+    packName: slack-bot_v2
+    prevname: ''
+    propagationLabels:
+    - all
+    toServerVersion: ''
+description: Integrate your organization's Slack app; using XSOAR as the backend
+display: SlackBot
+name: SlackBot
+script:
+  dockerimage: akoppadpalo/slack_bolt:v2
```

Comment:
If you could open a PR for the docker image in https://github.com/demisto/dockerfiles, we can move the docker image there.

Reply:
Yup I am on it. I will get it done this week